bibliopixel/cli/__command__.py

```diff
@@ -0,0 +1,71 @@
+class Command(object):
```

Comment:
I can't find any document of `__command__.py` either...?

Reply:
`__command__.py` was more of a placeholder for lack of a better name.
The Command object might be better off in the `__init__.py` of the cli folder.